clients/drcachesim/reader/reader.h

```diff
@@ -165,6 +165,11 @@ class reader_t : public reader_base_t {
     {
         return in_kernel_trace_;
     }
+    uint64_t
+    get_next_trace_pc() const override
```

Comment:
This can be in reader_base_t. No reason to make both reader_t and record_reader_t override it with identical definitions.

Reply:
Done.